src/Components/Header.jsx

```diff
@@ -1,6 +1,45 @@
+import "./Styles/Header.css";
+import { Link } from "react-router-dom";
+
 export default function Header() {
 
-    return (<div>
-        <h1>This is Header</h1>
-    </div>)
+    return (
+        <>
+        <nav>
+            <div className="nav-am flex-center-sb">
+                <Link className="logo" to="/"><h1 className="text-xl">Insta<span style={{color:"var(--insta-yellow)"}}>Fresh</span></h1></Link>    
```

Comment:
Try enclosing this color style in a single class you can call , instead of inline styling it

Reply:
Ok, I'll do that